### ISSUE TITLE
inner function malforms injected code 

### ISSUE THREAD

**Issue by shairubin** [shairubin/fun_with_ast#309]
def ref_lambda_rank_loss():
 strudel.info('Function start def default') # STRUDEL_TRACE_0
 def log_sigm(x):
   strudel.info('Function start def default') # STRUDEL_TRACE_0
   return -np.log(1 + np.exp(-x))
**# if you try to add code here identation will be broken** 
 dy = np.zeros(n)
 loss = 0
